### PR TITLE
[MSGINA] Update Portuguese (pt-PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
## Purpose

Update and improve pt-PT translation

### CODE REVIEW

**Comment by JoachimHenze** [comment]
Personally I don't see any gain or need for this PR. I don't see anything that would be better now compared to how it was before. I will not approve this one, but I will also not actively reject it. If anybody sees this as a win, feel free to handle it.

**Comment by zecarlos1957** [comment]
@JoachimHenze 
I don't understand your comment. I translated some English words into Portuguese, for example (English) Password to (Portuguese) Palavra-passe, and some others improvements.

**Review by HBelusca — APPROVED** [review]

File: dll/win32/msgina/lang/pt-PT.rc [hunk 1]
```diff
@@ -214,31 +214,31 @@ BEGIN
...
     IDS_INVALIDWORKSTATION "A sua conta está configurada para impedir que use este computador. Por favor, tente outro computador."
     IDS_ACCOUNTRESTRICTION "Não é possível iniciar sessão devido a uma restrição de conta."
-    IDS_EMERGENCYLOGOFFTITLE "Log Off ReactOS"
-    IDS_EMERGENCYLOGOFF "Click OK to immediately log off. Any unsaved data will be lost. Use this only as a last resort."
-    IDS_EMERGENCYRESTARTTITLE "Emergency restart"
-    IDS_EMERGENCYRESTART "Click OK to immediately restart. Any unsaved data will be lost. Use this only as a last resort."
+    IDS_EMERGENCYLOGOFFTITLE "Encerrar sessão no ReactOS"
+    IDS_EMERGENCYLOGOFF "Clique em OK para encerrar a sessão imediatamente. Quaisquer dados não guardados serão perdidos. Utilize isto apenas como último recurso."
+    IDS_EMERGENCYRESTARTTITLE "Reinício de emergência"
+    IDS_EMERGENCYRESTART "Clique em OK para reiniciar imediatamente. Quaisquer dados não guardados serão perdidos. Utilize isto apenas como último recurso.."
```
Comment (HBelusca) [inline dll/win32/msgina/lang/pt-PT.rc:241]:
```suggestion
    IDS_EMERGENCYRESTART "Clique em OK para reiniciar imediatamente. Quaisquer dados não guardados serão perdidos. Utilize isto apenas como último recurso."
```

**Review by HBelusca — CHANGES_REQUESTED** [review]
Missed that one.

**Review by HBelusca — APPROVED** [review]

File: dll/win32/msgina/lang/pt-PT.rc [hunk 2]
```diff
@@ -88,19 +88,19 @@ END
...
     LTEXT "Início de sessão para:", IDC_STATIC, 7, 78, 78, 8
     COMBOBOX IDC_CHANGEPWD_DOMAIN, 90, 75, 127, 144, CBS_DROPDOWNLIST | CBS_SORT | WS_TABSTOP
-    LTEXT "Password antiga:", IDC_STATIC, 7, 95, 78, 8
+    LTEXT "Palavra-passe antiga:", IDC_STATIC, 7, 95, 78, 8
     EDITTEXT IDC_CHANGEPWD_OLDPWD, 90, 92, 127, 12, ES_AUTOHSCROLL | ES_PASSWORD
-    LTEXT "Nova Password:", IDC_STATIC, 7, 111, 78, 8
+    LTEXT "Nova palavra-passe:", IDC_STATIC, 7, 111, 78, 8
     EDITTEXT IDC_CHANGEPWD_NEWPWD1, 90, 109, 127, 12, ES_AUTOHSCROLL | ES_PASSWORD
-    LTEXT "Confirme a nova Password:", IDC_STATIC, 7, 127, 78, 8
+    LTEXT "Confirme a nova palavra-passe:", IDC_STATIC, 7, 127, 78, 8
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/pt-PT.rc:103]: I do suggest to prove with a screenshot from reshacker, that this string will fit the 78 width. My gutfeeling is, that it won't.

Comment (zecarlos1957) [reply dll/win32/msgina/lang/pt-PT.rc]: Good shot. Thanks.

**Review by HBelusca — APPROVED** [review]
good to go with @JoachimHenze 's comment and your fix.

**Pull request merged**